modules/claims_api/app/swagger/claims_api/forms/form_2122_response_swagger.rb

```diff
@@ -220,7 +220,7 @@ class Form2122ResponseSwagger
                 key :type, :object
                 property :veteran do
                   key :type, :string
-                  key :description, 'Base64 encoded png image of the veterans signature'
+                  key :description, 'Base64 encoded png image of the veteran or claimant signature'
```

Comment:
You should prob put this back to how it was.
I'm pretty certain this is not being referenced anywhere, but we should still only change the file that needs changing.